src/Aspire.Hosting.Azure.ServiceBus/AzureServiceBusExtensions.cs

```diff
@@ -264,20 +253,62 @@ public static IResourceBuilder<AzureServiceBusResource> RunAsEmulator(this IReso
                 .WithImageRegistry(ServiceBusEmulatorContainerImageTags.AzureSqlEdgeRegistry)
                 .WithEndpoint(targetPort: 1433, name: "tcp")
                 .WithEnvironment("ACCEPT_EULA", "Y")
-                .WithEnvironment("MSSQL_SA_PASSWORD", password);
+                .WithEnvironment(context =>
+                {
+                    context.EnvironmentVariables["MSSQL_SA_PASSWORD"] = passwordParameter;
+                });
 
         builder.WithAnnotation(new EnvironmentCallbackAnnotation((EnvironmentCallbackContext context) =>
         {
             var sqlEndpoint = sqlEdgeResource.Resource.GetEndpoint("tcp");
 
             context.EnvironmentVariables.Add("ACCEPT_EULA", "Y");
             context.EnvironmentVariables.Add("SQL_SERVER", $"{sqlEndpoint.Resource.Name}:{sqlEndpoint.TargetPort}");
-            context.EnvironmentVariables.Add("MSSQL_SA_PASSWORD", password);
+            context.EnvironmentVariables.Add("MSSQL_SA_PASSWORD", passwordParameter);
         }));
 
         ServiceBusClient? serviceBusClient = null;
         string? queueOrTopicName = null;
 
+        var lifetime = ContainerLifetime.Session;
+
+        // Create a default file mount. This could be replaced by a user-provided file mount.
+        var configHostFile = Path.Combine(Directory.CreateTempSubdirectory("AspireServiceBusEmulator").FullName, "Config.json");
```

Comment:
Do we always want to create the directory up front? In the case where the lifetime of is persistent, we may not want this right?